src/neo4j_genai/components/embedder.py

```diff
@@ -22,6 +22,20 @@ class TextChunkEmbedder(Component):
 
     Args:
         embedder (Embedder): The embedder to use to create the embeddings.
+
+    Example:
+
+    .. code-block:: python
+
+        from neo4j_genai.components.embedder import TextChunkEmbedder
+        from neo4j_genai.embeddings.openai import OpenAIEmbeddings
+        from neo4j_genai.pipeline import Pipeline
+
+        embedder = OpenAIEmbeddings(model="text-embedding-3-large")
+        chunk_embedder = TextChunkEmbedder(embedder)
```

Comment:
Components can also be run independently from a pipeline, so in this example we could have:
```
asyncio.run(chunk_embedder.run(...))
```

Reply:
True I thought it would make sense for the example to be using it in a pipeline though I as imagine that's how most people will want to use it